md/release-notes.md

```diff
@@ -20,10 +20,16 @@ This new resource can be installed and updated on a Bonita platform while BPM se
 
 <a id="save-form"/>
 
-### Save Form (starting with Teamwork edition)
-When a form contains lots of fields, or when the user misses some information required to fill out the form at once, saving the form to complete it later is a need. This is now possible with the new "Save" widget, that can be added to any form.
+### Save a Form temporarily (starting with Teamwork edition)
+When a form contains lots of fields, or when the user misses some information required to fill out the form at once, saving the form to complete it later is a need. This is now possible with the new "Save Button" widget, that can be added to any form.
 JavaScript information entered by the user is temporarily saved in the browser local storage while the task is still in "pending" status. When the user comes back to the task, the values entered are displayed on the form, missing data can be added, so the user efficiently submits the form to either start a process instance or execute a task.
 
+<a id="new-case-overview"/>
+
+### New case overview (in the Community edition)
```

Comment:
~in the Community edition~ => all editions

Reply:
ok